tests/Keboola/Writer/LogTest.php

```diff
@@ -122,10 +121,6 @@ public function testDropLock()
         foreach ($this->logHandler->getRecords() as $logHandler) {
             $this->assertArrayHasKey('message', $logHandler);
 
-            if (strpos($logHandler['message'], 'is locked by 1 transactions') === false) {
-                continue;
-            }
```

Comment:
opravdu ten message nema vevnic `$table['dbName']`?  proc to smazes?

Reply:
dal jsem to pryč, protože message už neobsahuje `is locked by 1 transactions`